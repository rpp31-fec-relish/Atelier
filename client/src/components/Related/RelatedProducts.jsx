import React from 'react';

const RelatedProducts = (props) => {
  let product_id = props.product_id;

  return (
    <div id="related">
      <h4>Related Products</h4>
      <div id="related-products">
        <input type="image" alt="image"></input>
        <h6>Category</h6>
        <h6>Title</h6>
        <h6>Price</h6>
        <h6>Rating</h6>
      </div>
    </div>
  )

}

export default RelatedProducts;

