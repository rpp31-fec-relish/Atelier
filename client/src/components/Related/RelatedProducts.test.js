import RelatedProducts from './RelatedProducts.jsx';

import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';

Enzyme.configure({ adapter: new Adapter() });

describe('Related Products', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<RelatedProducts />);
    expect(wrapper).toMatchSnapshot();
  });
});