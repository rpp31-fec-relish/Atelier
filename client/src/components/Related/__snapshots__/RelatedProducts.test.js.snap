// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Related Products renders correctly 1`] = `
<div
  id="related"
>
  <h4>
    Related Products
  </h4>
  <div
    id="related-products"
  >
    <input
      alt="image"
      type="image"
    />
    <h6>
      Category
    </h6>
    <h6>
      Title
    </h6>
    <h6>
      Price
    </h6>
    <h6>
      Rating
    </h6>
  </div>
</div>
`;
